app/views/admin/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<% origin = '' %>
+
+<h1>Requests</h1>
+<div class="admin-requests">
+
+<% for request in @requests %>
```

Comment:
For loops are okay, but typically the syntax used for this kind of think in ruby would be

```ruby
@requests.each do |request|
   ...
end